packages/vector/collections.py

```
@@ -0,0 +1,303 @@
+                "hnsw_m": hnsw_config.m,
+            },
+        )
+
+    except KeyError as e:
+        logger.error(
+            "Collection config missing required parameter",
+            extra={
+                "correlation_id": correlation_id,
+                "missing_key": str(e),
+            },
+        )
```

Ruff (0.14.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)